### PR TITLE
update ReadMe

### PR DESCRIPTION
**Pull request by jjcrab**
Fixes issue #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,50 @@
...
+### Start the web-server
+
 ```
 uvicorn main:app --reload --host 0.0.0.0
 ```
 
-Get your PC's IP address from `ipconfig`. Go to <ipaddr>:8000 on your phone's browser.
+### Get your computer's IP address:
+
+- Windows users: type following command in your CMD prompt
```
Comment (shubham1172) [inline README.md:32]: nit:
```suggestion
- Windows (CMD/PowerShell)
```

File: README.md [hunk 2]
```diff
@@ -1,29 +1,50 @@
...
+```
+ifconfig
+```
+
+### Connect to your phone
+
+Go to `ip_address:8000` on your phone's browser. (Replace ip_address with your computer's IP address')
 
-**Note**: Make sure that your media player is an active window.
+### **Note**: Make sure that your media player is an active window.
```
Comment (shubham1172) [inline README.md:48]: Should this be a heading? Maybe we can add another header `Start using Remote-play` with this note and the screenshot.

File: README.md (hunk 1 above)
Comment (shubham1172) [inline README.md:32]: nit
```suggestion
- Windows users: type the following command in your CMD prompt
```

File: README.md [hunk 3]
```diff
@@ -1,29 +1,50 @@
...
-Get your PC's IP address from `ipconfig`. Go to <ipaddr>:8000 on your phone's browser.
+### Get your computer's IP address:
+
+- Windows users: type following command in your CMD prompt
+
+```
+ipconfig
+```
+
+- Linux/Mac users: type following command in your terminal
```
Comment (shubham1172) [inline README.md:38]: nit:
```suggestion
- Linux/Mac users: type the following command in your terminal
```

**Review by shubham1172 — CHANGES_REQUESTED** [review]
LGTM overall, thanks for the PR!

File: README.md (hunk 2 above)
Comment (jjcrab) [reply README.md:48]: Good idea. I will change this.

**Review by shubham1172 — APPROVED** [review]

**Comment by shubham1172** [comment]
Thanks again, @jjcrab 🎉

**Pull request merged**